src/main/proto/ga4gh/variants.proto

```diff
@@ -191,4 +191,29 @@ message Variant {
   // determination of genotype with respect to this variant. `Call`s in this
   // array are implicitly associated with this `Variant`.
   repeated Call calls = 12;
+
+  // The "variant_type" is used to denote e.g. structural variants.
+  // Examples:
+  //   DUP  : duplication of sequence following "start"; not necessarily in situ
+  //   DEL  : deletion of sequence following "start"
+  string variant_type = 13;
```

Comment:
@david4096 I actually had offered this as 2 ways to treat the problem, not as additive solution. You'll still have to document the allowed values in the OT object...